ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_3.ipynb

```diff
@@ -467,7 +467,7 @@
    "id": "77de4703-5400-4a4b-a928-0af8878c330a",
    "metadata": {},
    "source": [
-    "Although the $p$-value is does not conclusively reject the null hypothesis at all reasonable confidence levels, it does suggest that we might want to relax the residual normality assumption. `bootstrap` makes no such assumption about the residuals, and it can go beyond the standard errors, producing bias-corrected confidence intervals. The standard errors produced by `bootstrap` matche those produced by `linregress` fairly well; however `linregress` may overestimate these quantities for this data."
+    "Although the $p$-value is not sufficient to conclusively reject the null hypothesis at all reasonable confidence levels, it does suggest that we might want to relax the residual normality assumption. `bootstrap` makes no such assumption about the residuals, and it can go beyond the standard errors, producing bias-corrected confidence intervals. The standard errors produced by `bootstrap` match those produced by `linregress` fairly well; however `linregress` may overestimate these quantities for this data."
```

Comment:
""not sufficient" suggests to me that the $p$-value is "not large enough", which is the opposite of what we would want to say.
```suggestion
    "Although the $p$-value is too high to conclusively reject the null hypothesis at all reasonable confidence levels, it does suggest that we might want to relax the residual normality assumption. `bootstrap` makes no such assumption about the residuals, and it can go beyond the standard errors, producing bias-corrected confidence intervals. The standard errors produced by `bootstrap` match those produced by `linregress` fairly well; however `linregress` may overestimate these quantities for this data."
```

Reply:
Interesting, I wouldn't have thought of it like that. Yes, it's better to be extra clear here. I like "is not small enough to" or "is still too high to", implying that the p-value is still somewhat small, just not small enough.